components/Navbar/Navbar.tsx

```diff
@@ -22,25 +22,31 @@ export const Navbar: React.FC = ({ children }) => {
           the signup page, where they shouldn't be displayed
           */
 
-                  href="home"
                   className={router.pathname == "/home" ? styles.clicked : styles.navlink}
+                  onClick={() => {
+                    router.push("/home");
+                  }}
```

Comment:
why arent we using href's anymore? Not questioning, just wondering the reason for the change

Reply:
updated for consistency with my other menu but either way works -- used the push bc i was appending to urls but now it doesn't _really_ make a difference

Reply:
ah ok. I think thats ok for now. if justin says something during code review ill change it then